### PR TITLE
Migrates zipkin-kafka docker image

### PR DESCRIPTION
**Pull request by codefromthecrypt**
See https://github.com/openzipkin/docker-zipkin/issues/86
cc @jeqo

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -22,6 +22,20 @@ nginx. To build, run something like
...
+
+We also provide an image that runs both Kafka+ZooKeeper for testing the [Kafka collector](https://github.com/openzipkin/zipkin/tree/master/zipkin-collector/kafka)
+and the upcoming [Kafka storage](https://github.com/openzipkin-contrib/zipkin-storage-kafka).
+To build, run something like
+
+```bash
+$ docker build -t openzipkin/zipkin-kafka:test -f docker/kafka/Dockerfile .
+```
+
+Then configure the [Kafka sender](https://github.com/openzipkin/zipkin-reporter-java/blob/master/kafka11/src/main/java/zipkin2/reporter/kafka11/KafkaSender.java) using a `bootstrapServers` value of `host.docker.internal:9092` if your application is inside the same docker network or `localhost:19092` if not, but running on the same host.
```
Comment (codefromthecrypt) [inline docker/README.md]:
```suggestion
Then configure the [Kafka sender](https://github.com/openzipkin/zipkin-reporter-java/blob/master/kafka/src/main/java/zipkin2/reporter/kafka/KafkaSender.java) using a `bootstrapServers` value of `host.docker.internal:9092` if your application is inside the same docker network or `localhost:19092` if not, but running on the same host.
```

File: docker/hooks/post_push [hunk 2]
```diff
@@ -3,14 +3,13 @@
 set -v
 
 if [[ "$DOCKER_REPO" == "index.docker.io/openzipkin/zipkin" ]]; then
-  # We always push zipkin-slim / zipkin-ui
-  echo Pushing zipkin-slim and zipkin-ui
+  echo Pushing test zipkin-slim and test images
```
Comment (codefromthecrypt) [inline docker/hooks/post_push]:
```suggestion
  echo Pushing zipkin-slim and test images
```

File: docker/hooks/post_build [hunk 3]
```diff
@@ -15,16 +15,22 @@ for tag in ${TAGS[@]:1}; do
...
 done
 
+echo Building zipkin-kafka
+docker build --build-arg version="$SOURCE_BRANCH" -f "docker/kafka/Dockerfile"-t "openzipkin/zipkin-kafka:${TAGS[0]}" --target zipkin-kafka .
+for tag in ${TAGS[@]:1}; do
+  docker tag "openzipkin/zipkin-kafka:${TAGS[0]}" "openzipkin/zipkin-kafka:$tag"
+done
+
 # We also build storage images to correspond with the server version to keep schemas up to date
 for storage in cassandra; do
```
Comment (anuraaga) [inline docker/hooks/post_build]: Should be able to just add kafka here I think

File: docker/README.md [hunk 4]
```diff
@@ -22,6 +22,20 @@ nginx. To build, run something like
 $ docker build -t openzipkin/zipkin-ui:test -f docker/Dockerfile --target zipkin-ui .
 ```
 
+## zipkin-kafka Docker image
```
Comment (anuraaga) [inline docker/README.md]: Can you either move the Cassandra readme here or this to the subdirectory? Either is fine if consistent

**Comment by codefromthecrypt** [comment]
good idea

File: docker/hooks/post_build (hunk 3 above)
Comment (codefromthecrypt) [reply docker/hooks/post_build]: yep

File: docker/README.md (hunk 4 above)
Comment (codefromthecrypt) [reply docker/README.md]: done

File: docker/README.md [hunk 5]
```diff
@@ -1,26 +1,32 @@
...
 
 ```bash
 $ docker build -t openzipkin/zipkin:test -f docker/Dockerfile .
 ```
 
-If you want the slim distribution instead, run something like
+If you want the slim distribution instead, run:
 
 ```bash
 $ docker build -t openzipkin/zipkin:test -f docker/Dockerfile . --target zipkin-slim
```
Comment (codefromthecrypt) [inline docker/README.md]:
```suggestion
$ docker build -t openzipkin/zipkin-slim:test -f docker/Dockerfile . --target zipkin-slim
```

**Pull request merged**